plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/applicationexplorer/CreateComponentWizardPage.java

```diff
@@ -155,54 +132,31 @@ protected void doCreateControls(Composite parent, DataBindingContext dbc) {
 		Label componentTypesLabel = new Label(parent, SWT.NONE);
 		componentTypesLabel.setText("Component type:");
 		GridDataFactory.fillDefaults().align(SWT.FILL, SWT.TOP).applyTo(componentTypesLabel);
-		Tree componentTypesTree = new Tree(parent, SWT.SINGLE | SWT.BORDER);
+		org.eclipse.swt.widgets.List componentTypesList = new org.eclipse.swt.widgets.List(parent, SWT.SINGLE | SWT.BORDER);
```

Comment:
When creating a component, i do not see what type is selected by alizer in the list (used quarkus). there is no scrolling neither .

Reply:
Will fix the scrollbar issue but can't reproduce the default selection from alizer (tested with a Quarkus and SpringBoot projects). Make sure your are using registry.devfile.io in your settings